site-packages/integralstor/zfs.py

```diff
@@ -1798,6 +1798,27 @@ def get_pool_usage(pool_name):
         return d, None
 
 
+def get_pool_events_history(pool_name, action):
```

Comment:
Perhaps better to not club events and history both under one function? Changes are easier if we have separate functions and templates for such actions. But if we feel that this might lead to many disparate functions, we can have one function called `get_pool_actions(pool_name, action)`(might eventually grow longer with all the conditional checks) and have separate templates for each action.